From efa39292876bb0bdceec1eb328f1ad77adc21301 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 25 Aug 2016 20:43:33 +0800
Subject: [PATCH 1135/5242] MLK-13119-1 ARM: dts: imx6q: add vpu clock for cpu
 frequency

commit  79d814f9ba39edfcf2602230ee671c330211fcaa from
https://source.codeaurora.org/external/imx/linux-imx.git

When VPU is running at 396MHz, need to increase
VDDSOC_CAP voltage for all cpu set-points, so add
vpu clock node for cpufreq driver.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6q.dtsi |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6q.dtsi b/arch/arm/boot/dts/imx6q.dtsi
index 818999b..0e6eb32 100644
--- a/arch/arm/boot/dts/imx6q.dtsi
+++ b/arch/arm/boot/dts/imx6q.dtsi
@@ -45,10 +45,12 @@
 				 <&clks IMX6QDL_CLK_PLL1_SYS>,
 				 <&clks IMX6QDL_CLK_PLL1>,
 				 <&clks IMX6QDL_PLL1_BYPASS>,
-				 <&clks IMX6QDL_PLL1_BYPASS_SRC>;
+				 <&clks IMX6QDL_PLL1_BYPASS_SRC>,
+				 <&clks IMX6QDL_CLK_VPU_AXI_PODF>;
 			clock-names = "arm", "pll2_pfd2_396m", "step",
 				      "pll1_sw", "pll1_sys", "pll1",
-				      "pll1_bypass", "pll1_bypass_src";
+				      "pll1_bypass", "pll1_bypass_src",
+				      "vpu_axi_podf";
 			arm-supply = <&reg_arm>;
 			pu-supply = <&reg_pu>;
 			soc-supply = <&reg_soc>;
-- 
1.7.9.5

